src/Settings.php

```diff
@@ -290,6 +292,10 @@ public function display_types_table() { // phpcs:ignore Generic.Metrics.NestingL
 					}
 
 				endif;
+
+					// phpcs:disable WPForms.Comments.PHPDocHooks.RequiredHookDocumentation
+					do_action( 'fileuploadtypes_settings_display_types_table_after_enabled_types' );
```

Comment:
`awesomemotive/wpforms-phpcs` package has an issue and notify about an invalid hook name incorrectly. For better readability the `FileUploadTypes` namespace should be converted to `file_upload_types`. 

Could you please change nabes for all new hooks?
Obviously, you will have to use `phpcs:disable` / `phpcs:enable` for reducing a noise.

Reply:
done